packages/@lwc/integration-karma/test/accessibility/synthetic-cross-root-aria/index.spec.js

```diff
@@ -244,23 +251,17 @@ if (!process.env.NATIVE_SHADOW) {
                 elm2.linkUsingBoth({ idPrefix: 'prefix-2' }); // ids must be globally unique
             }).toLogWarningDev(expectedMessageForCrossRoot);
 
-            // dispatcher is still called twice
-            expect(dispatcher.calls.allArgs()).toEqual([
-                [
-                    'CrossRootAriaInSyntheticShadow',
-                    {
-                        tagName: 'x-aria-source',
-                        attributeName: 'aria-labelledby',
-                    },
-                ],
-                [
-                    'CrossRootAriaInSyntheticShadow',
-                    {
-                        tagName: 'x-aria-source',
-                        attributeName: 'aria-labelledby',
-                    },
-                ],
-            ]);
+            // dispatcher is still called twice for CrossRootAriaInSyntheticShadow
+            expect(
+                dispatcher.calls
+                    .allArgs()
+                    .filter(
+                        (dispatch) =>
+                            dispatch[0] === 'CrossRootAriaInSyntheticShadow' &&
+                            dispatch[1].tagName === 'x-aria-source' &&
+                            dispatch[1].attributeName === 'aria-labelledby'
+                    )
+            ).toHaveSize(2);
```

Comment:
I think this is less clear than the previous (equivalent) code. I would prefer to change it back, but I also don't have a really strong opinion, since it's just test code.

Reply:
This variation is less strict about whether the dispatcher includes calls for other events. The previous code broke with the addition of new reporting events, while this one would not.

I'm fine changing it back since it's less critical to have that distinction with the new helper function. The previous behavior would still be asserting that the dispatcher is called twice for `CrossRootAriaInSyntheticShadow` AND zero times for `NonStandardAriaReflection`, but that's behavior less likely to break in the future.

Reply:
Yep that's exactly what I was thinking – let's use the new `attachReportingControlDispatcher` to control the filtering, rather than doing it here.